test/ClientProxyTest.js

```diff
@@ -52,9 +52,9 @@ describe('Generic proxy test', function () {
         assert.equal(mapProxy.getConnectedServerVersion(), -1);
     });
 
-    it('Client with a 3.7 server connection should return the version', function () {
```

Comment:
nit: it makes sense to leave 3.7 version here as this branch is only compatible with IMDG 3.x.